content/en/docs/reference/scheduling/config.md

```diff
@@ -255,7 +255,183 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
-## {{% heading "whatsnext" %}}
+### MultiPoint config
+
+There is an additional field in the profile config, `MultiPoint`, which allows 
+for easily enabling or disabling a plugin across several extension points. The intent 
+of `MultiPoint` config is to simplify the configuration needed for users and administrators 
+when using custom profiles.
+
+Consider a plugin, `MyPlugin`, which implements the `PreScore`, `Score`, `PreFilter`, 
+and `Filter` extension points. To enable `MyPlugin` for all its available extension 
+points, the profile config simply looks like:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: multipoint-scheduler
+    plugins:
+      multiPoint:
+        enabled:
+        - name: 'MyPlugin'
+```
+
+This would equate to manually enabling `MyPlugin` for all of its extension 
+points, like so:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: non-multipoint-scheduler
+    plugins:
+      preScore:
+        enabled:
+        - name: 'MyPlugin'
+      score:
+        enabled:
+        - name: 'MyPlugin'
+      preFilter:
+        enabled:
+        - name: 'MyPlugin'
+      filter:
+        enabled:
+        - name: 'MyPlugin'
+```
+
+Specific extension points can be excluded from `MultiPoint` expansion using 
+the `disabled` fields. This works with disabling default plugins, out-of-tree plugins, 
+or with the wildcard (`'*'`) to disable all plugins. An example of the above to 
+disable `Score` and `PreScore` would be:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: non-multipoint-scheduler
+    plugins:
+      multiPoint:
+        enabled:
+        - name: 'MyPlugin'
+      preScore:
+        disabled:
+        - name: '*'
+      score:
+        disabled:
+        - name: '*'
+```
+
+In `v1beta3`, all default plugins are enabled internally through `MultiPoint`. 
+However, individual extension points are still available to allow flexible 
+reconfiguration of the default values (such as ordering and Score weights). For 
+example, consider two Score plugins `DefaultScore1` and `DefaultScore2`, each with 
+a weight of `1`. They can be reordered with different weights like so:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: multipoint-scheduler
+    plugins:
+      score:
```

Comment:
multiPoint? Order only really matters for Filter

Reply:
Yeah I was trying to show a couple things here, that we can reorder *and* re-weight the plugins. Order technically only matters for Filter, true, but I think this still gets both of those points across

Reply:
sounds good